src/docs/BUILD.md

```diff
@@ -29,6 +29,9 @@
     cmake --install ./build/ --config Release --prefix ov
     ```
 
+> **NOTE**: There is a known Python API issue with `ov::Tensor`. The issue is reproduced when building OpenVINO GenAI from sources while using OpenVINO from archives. Using `ov::Tensor` with OpenVINO GenAI will fail on Windows and Linux Systems. Possible errors: `TypeError: generate(): incompatible function arguments.`, `TypeError: __init__(): incompatible constructor arguments.`, `TypeError: Unregistered type : ov::Tensor`.
```

Comment:
Instructions updated.

Reply:
Why not link to wiki https://github.com/openvinotoolkit/openvino/wiki#how-to-build? It's less probable that it changes because they can move the file. If they move the file, ov is expected to update the link in the wiki. Same for installing.

Reply:
Linked build instructions to wiki.